tests/test_deprecated.py

```diff
@@ -155,3 +155,27 @@ def __init__(self, a: Any, b: Any, c: Any) -> None:
         expected_class_doc += "\n\n\n"
     assert decorated_class.__name__ == _Class.__name__
     assert decorated_class.__doc__ == expected_class_doc
+
+
+def test_deprecation_decorator_default_removed_version() -> None:
+    deprecated_version = "1.1.0"
+    decorator_deprecation = _deprecated.deprecated(deprecated_version)
+    assert callable(decorator_deprecation)
+
+    def _func(_: Any) -> int:
```

Comment:
Do you think the argument _ is required? It is not introduced by this PR but I'm just curious about it.

Reply:
You're right. The argument is not relevant to the test. I think we can fix that in a follow up, along with the other tests in this file as well as `test_experimental.py`.

Reply:
Thank you for your reply.

> I think we can fix that in a follow up, along with the other tests in this file as well as `test_experimental.py`.

I agree. Let me merge this PR.

Reply:
https://github.com/optuna/optuna/pull/1423